Sources/MountebankSwift/Documentation.docc/MountebankSwift.md

```diff
@@ -7,3 +7,46 @@ A swift client library for Mountebank.
 MountebankSwift is a client library for [Mountebank](https://www.mbtest.org/) - open source tool that
 provides test doubles over the wire. It provides all the [api functionality](https://www.mbtest.org/docs/api/overview)
 to interact with a running Mountebank instance.
+
+## Topics
+
+### Connect to Mountebank
+- ``Mountebank``
+
+### Creating Imposters
+- ``Imposter``
+- ``Imposters``
+- ``NetworkProtocol``
+
+### Creating Stubs
+- ``Stub``
+
+### Creating Predicates
+- ``Predicate``
+- ``Request``
+- ``PredicateParameters``
+- ``JSONPath``
+- ``XPath``
```

Comment:
I'm doubting to just nest `JSONPath` and `XPath` inside of `PredicateParameters` to prevent them showing up in this list. 

(Any sybols you leave out will be automatically added to the bottom of the list)

Reply:
I do think it would be clearer to nest them. So the usage is more clear.

Reply:
Ok, lets do that